src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1558,18 +1558,33 @@ namespace winrt::TerminalApp::implementation
         DataPackage dataPack = DataPackage();
         dataPack.RequestedOperation(DataPackageOperation::Copy);
 
-        // copy text to dataPack
-        dataPack.SetText(copiedData.Text());
+        // The EventArgs.Formats() is an override for the global setting "copyFormatting"
+        //   if and only if it is not set
+        // Since 'Formats' cannot be represented as an optional in the EventArgs,
+        //   a sentinel value of 0 represents that "Formats" was not set.
```

Comment:
i'm sorry? 0 means "none". It can't be represented as an optional?
Isn't that what the IReference is?

Isn't "plain" the same as none? wait a second

Reply:
wait just a second, what does it mean to copy NO formats?

did you have to make plain 1 so that you could use 0 as a sentinel value when you already have an option type?

Reply:
Ah. This comment was also left over from a former impl that did _not_ use IReference. Now we don't need a random sentinel value.

The current implementation has that you can suppress plain text by omitting "plain". So "none" really is "don't copy anything". I can't see a use case for this but I guess it's correct.

Should I change the behavior to always include the plain text? Like this...
- none/plain --> copy plain text
- html/rtf --> copy html/rtf data (respectively), but always include the plain text

Reply:
In that case, the following scenarios all mean the same thing...
- `[]`
- `none`
- `plain`
I guess I could remove `none` as an accepted string and only allow `plain`. Whatcha think?

Reply:
The spec was not very clear on this issue. But I implemented it as follows:
- `none` = `plain` = `[]` = all clear